core/domain/feedback_services.py

```diff
@@ -291,12 +308,39 @@ def _add_feedback_message_reference(user_id, reference):
         enqueue_feedback_message_email_task(user_id)
 
 
-def send_feedback_message_email(exploration_id, thread_id, message_id):
+def update_feedback_email_retries(user_id):
+    model = feedback_models.UnsentFeedbackEmailModel.get(user_id)
+    time = (datetime.datetime.utcnow() - model.created_on).seconds
```

Comment:
Also maybe rename to time_since_creation_secs or model_age_secs or something that's more specific than just 'time'...